ext/crypto/lib.rs

```diff
@@ -1272,7 +1294,7 @@ pub async fn op_crypto_import_key(
           })?;
 
           if bytes_consumed
-            != rsa::pkcs1::der::Length::new(pk_info.private_key.len() as u16)
+            != spki::der::Length::new(pk_info.private_key.len() as u16)
```

Comment:
Just double-checking: these are the same thing, right? (`Length` from the der crate.)